tests/testthat/test-plot.R

```diff
@@ -65,3 +65,14 @@ test_that("basic plot test, spheres", {
     }
   )
 })
+
+test_that("rgplot() works", {
+  withr::local_envvar(RGL_USE_NULL = TRUE)
```

Comment:
https://stackoverflow.com/a/46320771/5489251

Reply:
Should this be a comment in the code instead? Only what's in the code will remain visible later.